src/nouveau/src/nouveau_fabric_search.erl

```diff
@@ -39,13 +39,20 @@ go(DbName, #doc{} = DDoc, IndexName, QueryArgs0) ->
     Shards = get_shards(DbName, QueryArgs0),
     {PackedBookmark, #{limit := Limit, sort := Sort} = QueryArgs1} =
         maps:take(bookmark, QueryArgs0),
+    TopN = maps:get(top_n, QueryArgs1, 10),
```

Comment:
If you want could pull the `?TOP_N_DEFAULT` into a define just to keep the numeric constants at the top of the module. It's more verbose but it's the pattern we use often in other modules.